english/story-continues

```diff
@@ -0,0 +1 @@
+The story continues.  You wake up and it was all a dream.  Then, in the corner of your eye you see [a] (http://ancientexplorers.com/wp-content/uploads/sites/6/2015/12/sparkling-blue-universe_131922.jpg)  
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.